### PR TITLE
Create samsteingraber.md

### PR DESCRIPTION
**Pull request by Steingraber**

### CODE REVIEW

File: samsteingraber.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+I learned how to swim as an adult
+I love the smell of eggs
+I can solve a rubiks cube
```
Comment (JoshuaToenyes) [inline samsteingraber.md:3]: I don't believe the Rubik's Cube...

Comment (pcook21) [reply samsteingraber.md:3]: #3 Is the Lie

Comment (scharfek) [reply samsteingraber.md:3]: #2 is the lie

Comment (mckennaheasley) [reply samsteingraber.md:3]: #3 is the lie. You can't solve a rubiks cube?

Comment (ruggedstickman) [reply samsteingraber.md:3]: swimming

**Pull request merged**